### ISSUE TITLE
Accessibility android - label gesture recognizer doesn't not work when TalkBack is on 

### ISSUE THREAD

**Issue by pamela032709** [xamarin/Xamarin.Forms#5375]
### Description
1.enable talkback on your android device 
When accessibility - talkback is on the images or labels gestures does not work 
to reproduce please follow this 
 <Label Text="Search"  AutomationProperties.IsInAccessibleTree="true" 
   AutomationProperties.HelpText="Double  tap search"
      HorizontalTextAlignment="Center"/>
        <Label.GestureRecognizers>
             <TapGestureRecognizer Tapped="get_my_info" NumberOfTapsRequired="1" />
             </Label.GestureRecognizers>
     

### Expected Behavior
when you tapped it raises the command and do what is suppose to do. the reader ignores the command so it only reads it as a label with no functionality. this doesn't not happen on ios  
### Basic Information

- Version with issue: Xamarin forms 3.5
- Last known good version:
- IDE:
- Platform Target Frameworks: 
  - iOS:  
  - Android:   all versions 
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices: samasung galaxy s8 , s9 , s10 and many more

**Comment by pamela032709** [comment]
![screen shot 2019-02-25 at 11 03 34 am](https://user-images.githubusercontent.com/19438031/53350475-1f987300-38ed-11e9-8b47-1cd46cb732b1.png)

**Comment by samhouts** [comment]
@pamela032709 Can you please attach your sample project? Thanks!

**Comment by samhouts** [comment]
@pamela032709  Since we haven't heard from you in more than 30 days, we hope this issue is no longer affecting you. If it is, please reopen this issue and provide the requested information so that we can look into it further. Thank you!
